### ISSUE TITLE
Add author blurb to site

### ISSUE THREAD

**Issue by egardner** [gettypubs/beyond-the-printed-page#4]
Create an `authors.yml` file in `_data` that serves as an author directory, and includes the following info:
- Author name
- Author thumbnail pic
- Short bio
- Email (optional)
- Twitter (optional)
- Website (optional)

If an article's metadata in `site.author` is `==` to an author in this index, the article template should output a block of author information:

``` js
{% if page.author %}
{% assign current_author = '' %}
  {% for author in site.data.authors %}
    {% if page.author == author.name %}
      {% assign current_author = author.name %}
      {% break %}
    {% endif %}
  {% endfor %}
{% endif %}
```

And then use the `current_author` value to pull bio, picture, etc.

**Comment by geealbers** [comment]
I updated the authors.yml file to include fairly complete details for all the authors in our sample issue.

Eric, can you take a look at the styling for the blurbs though? I think it could use a little more line space after the author name. In these examples both the website, and the twitter links look too tight to the name:
- http://gettypubs.github.io/beyond-the-printed-page/2014/11/26/the-future-of-digital-publishing-is/
- http://gettypubs.github.io/beyond-the-printed-page/2014/09/12/empathy-in-art-book-publishing/

And for when there is no author info in the yml file, let's leave it totally blank. The "By ___ ___" tag continues to feel awkward to me.

Thanks.

**Comment by egardner** [comment]
Fixed as of b0f5d7439decf99037334da92bacaf698beb3906
